pyccel/ast/cudaext.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#------------------------------------------------------------------------------------------#
+# This file is part of Pyccel which is released under MIT License. See the LICENSE file or #
+# go to https://github.com/pyccel/pyccel/blob/master/LICENSE for full license details.     #
+#------------------------------------------------------------------------------------------#
+"""
+CUDA Extension Module
+Provides CUDA functionality for code generation.
+"""
+from .internals      import PyccelFunction
+
+from .datatypes      import VoidType
+from .core           import Module, PyccelFunctionDef
+
+__all__ = (
+    'CudaSynchronize',
+)
+
+class CudaSynchronize(PyccelFunction):
+    """
+    Represents a call to  Cuda.deviceSynchronize for code generation.
+    
+    This class serves as a representation of a synchronization call to the CUDA.   
```

Comment:
" a synchronization call to the CUDA."

I don't understand this phrase

Reply:
[162c5e8](https://github.com/pyccel/pyccel-cuda/pull/42/commits/162c5e81e51779dd936a0566cecbdc03fe5efdc7)